Incorrect claim rule

piaudonn opened the pull request:
Line 99: => issue(Type = "http://schemas.microsoft.com/authorization/claims/permit", Value = "false"); will actually issue a Permit claim.
I suggest => issue(Type = "http://schemas.microsoft.com/authorization/claims/deny", Value = "true"); which will effectively issue a Deny claim and block the access at the ADFS level according to what the bullet point above is mentioning: "- Block Exchange ActiveSync by using Active Directory Federation Services (AD FS) rules."

azurecla commented:
Hi __@DuoKristina__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@DuoKristina : Thanks for your contribution to the Azure documentation! The author, @kgremban, has been notified to review your proposed change.

kgremban commented:
#sign-off

piaudonn committed:
Incorrect claim rule

Line 99: => issue(Type = "http://schemas.microsoft.com/authorization/claims/permit", Value = "false"); will actually issue a Permit claim.
I suggest => issue(Type = "http://schemas.microsoft.com/authorization/claims/deny", Value = "true"); which will effectively issue a Deny claim and block the access at the ADFS level according to what the bullet point above is mentioning: "- Block Exchange ActiveSync by using Active Directory Federation Services (AD FS) rules."
